tests/vrf/test_vrf.py

```diff
@@ -532,10 +532,13 @@ def test_show_bgp_summary(self, duthosts, rand_one_dut_hostname, cfg_facts):
             bgp_summary_string = duthost.shell("vtysh -c 'show bgp vrf {} summary json'".format(vrf))['stdout']
             bgp_summary = json.loads(bgp_summary_string)
 
-            for info in bgp_summary.itervalues():
-                for peer, attr in info['peers'].iteritems():
+            for info in bgp_summary:
+                for peer, attr in bgp_summary[info]['peers'].iteritems():
                     prefix_count = attr['prefixReceivedCount']
-                    assert int(prefix_count) == route_count, "%s should received %s route prefixs!" % (peer, route_count)
+                    if info == "ipv4Unicast" and attr['idType'] == 'ipv6':
+                        continue
```

Comment:
Why skip this checking? Should both 'ipv4' and ipv6' be checked?

Reply:
The problem is that ipv4 is default for all bgp peering's, and ipv6 peers appear under 'ipv4Unicast', they have value 'prefixReceivedCount' equal to 0 which leads to fail of test. Both ipv4 and ipv6 peers are being checked, but ipv4 peers are being checked under 'ipv4Unicast' and ipv6 under 'ipv6Unicast'. Meanwhile, ipv6 peers under 'ipv6Unicast' have expected value of 'prefixReceivedCount' equal to 6400.
We skip only ipv6 peers which are under 'ipv4Unicast' key, due to FRR default behavior.
Here is example of output: [7464](https://github.com/FRRouting/frr/issues/7464)